.github/actions/paths-filter/action.yml

```diff
@@ -20,6 +20,9 @@ outputs:
   client-common:
     value: ${{ steps.changes.outputs.client-common }}
     description: True if a files in the ionic-client part was edited.
+  news-fragments:
+    value: ${{ steps.changes.outputs.news-fragments }}
```

Comment:
should stay consistent with the name `newsfragments` without dash

Reply:
done